### DISCUSSION TITLE
Request for Assistance Regarding Account Suspension

### DISCUSSION THREAD

**Discussion by JennyT3** [forem/forem#21185] [Q&A]
@nmendybayev Dear Dev.to Support Team,

I hope this message finds you well.

I am reaching out because I have received a notification indicating that my account has been suspended, and my ability to post and comment has been restricted. I am unclear about the reason for this suspension and would like to understand more about the situation so I can address any issues and reactivate my account as soon as possible.

Could you please provide me with more details about the reason for the suspension? Additionally, are there any specific actions I need to take to resolve this issue?

Thank you in advance for your assistance. I look forward to your response so we can resolve this matter promptly.

Best regards,

Jenny T. 
<img width="652" alt="Captura de pantalla 2024-07-24 a las 14 32 10" src="https://github.com/user-attachments/assets/9678f703-f9f4-4eb9-8745-f47fc5e37730">

**Comment by nmendybayev** [comment]
Hi. I have been having the same problem for 2 months. I sent 10 emails, nobody replied.
